cmd/action/apply.go

```diff
@@ -14,9 +14,10 @@ import (
 var (
 	// ApplyFlags are the flags used in Apply command.
 	ApplyFlags struct {
-		DSN  string
-		File string
-		Web  bool
+		DSN     string
+		File    string
+		Web     bool
+		Address string
```

Comment:
I'd call it `Port` tbh

Reply:
I'd call it `--addr` as it's passed as-is to `Listen`.